handlers/LunchHandler.js

```diff
@@ -121,6 +140,89 @@ export default class LunchHandler extends BaseHandler {
 		);
 	}
 
+	getHotPotMenu() {
+		return new Promise((resolve, reject) => {
+			http.request(optionsPaevapraed, (res) => {
+				let item = '';
+
+				res.setEncoding('utf8');
+				res.on('data', (chunk) => {
+					const $ = cheerio.load(chunk);
+					const chunkHtml = $('#HOTPOT_FOOD').html();
+
+					if (chunkHtml) {
```

Comment:
Avoid casting to boolean, write the actual check (for example when it returns null if not found then if (chunkHtml !== null).